Patterns/Portuguese/Portuguese-DateTime.yaml

```diff
@@ -339,8 +339,10 @@ TimeRegex9: !nestedRegex
   # onze (e)? vinte e cinco
   def: \b(?<writtentime>{HourNumRegex}\s+({TensTimeRegex}\s*)(e\s+)?{MinuteNumRegex}?)\b
   references: [ HourNumRegex, TensTimeRegex, MinuteNumRegex ]
-#TimeRegex10: !simpleRegex
+TimeRegex10: !simpleRegex
```

Comment:
The more correct thing here would be to delete it also from the code in the other platforms.

Reply:
yes I did that now